src/SDKs/IotHub/IotHub.Tests/Helpers/IotHubTestUtilities.cs

```diff
@@ -21,6 +21,22 @@ public class IotHubTestUtilities
         public static string DefaultResourceGroupName = "DotNetHubRG";
         public static string DefaultUpdateResourceGroupName = "UpdateDotNetHubRG";
         public static string EventsEndpointName = "events";
+        public static string DefaultCertificateIotHubName = "DotNetCertificateHub";
```

Comment:
Might be better to programmatically generate this certificate.

Reply:
OpenSsl is not compatible with netcoreapp1.1